packages/fxa-content-server/app/scripts/views/settings/subscription.js

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+import Cocktail from 'cocktail';
+import FormView from '../form';
+import SettingsPanelMixin from '../mixins/settings-panel-mixin';
+import Template from 'templates/settings/subscription.mustache';
+
+const View = FormView.extend({
+  template: Template,
+  className: 'subscription',
+  viewName: 'settings.subscription',
+
+  events: {
+    'click button': 'submit',
+  },
+
+  initialize (options = {}) {
+    this._subscriptionUrl = options.config.subscriptionUrl;
+
+    this._featureFlags = Object.assign({}, {
+      enableManageButton: false
+    }, options.config.featureFlags.subscriptions || {});
+
+    this._config = Object.assign({}, {
+      allowedLanguages: ['en-US'],
+      enabled: false,
+      managementClientId: '98e6508e88680e1a',
+      managementScopes: 'profile https://identity.mozilla.com/account/subscriptions',
+      managementTokenTTL: 900,
+      managementUrl: 'http://127.0.0.1:3031',
+    }, options.config.subscriptions || {});
+  },
+
+  beforeRender () {
+    if (! this.supportSubscription()) {
+      this.remove();
+    }
+  },
+
+  submit () {
+    const {
+      managementClientId,
+      managementScopes,
+      managementTokenTTL,
+      managementUrl,
+    } = this._config;
+    const account = this.user.getSignedInAccount();
+    account
```

Comment:
Oh, also, wanted to note: This is not the final implementation for generating this token. Leaving #881 open to work out how to better generate and pass a token to the subscription management pages, probably involving a code exchange that enforces an IP address check to help ensure it's the same user navigating from FxA.